pymongo/uri_parser.py

```diff
@@ -469,8 +469,8 @@ def parse_uri(
             raise ConfigurationError(
                 'The "dnspython" module must be '
                 "installed to use mongodb+srv:// URIs. "
-                "To fix this error install pymongo with the srv extra:\n "
-                '%s -m pip install "pymongo[srv]"' % (python_path)
```

Comment:
I guess it doesn't hurt to keep this error just in case but is this even possible now that dnspython is a required dependency?